en/micro-integrator/docs/references/connectors/mongodb-connector/mongodb-connector-overview.md

```diff
@@ -0,0 +1,33 @@
+
+# MongoDB Connector Overview
+
+The MongoDB Connector allows you to connect to the MongoDB database via different connection URI and perform CRUD operations on the database.
+
+The supported connection URI types and connection options are listed in the [MongoDB Connection String](https://docs.mongodb.com/manual/reference/connection-string/) documentation.
+
+To see the available MongoDB Connector, navigate to the [connector store](https://store.wso2.com/store/assets/esbconnector/list) and search for "MongoDB".
+
+<img src="../../../../assets/img/connectors/mongodb-connector-store.png" title="MongoDB Connector Store" width="200" alt="MongoDB Connector Store"/>    
+
+## Compatibility
+
+| Connector version | Supported WSO2 ESB/EI version |
```

Comment:
Let's mention with which MongoDB version connector compatibility is tested with. 

"This connector is tested with MongoDB version xxxx"

Reply:
Fixed with the latest commit https://github.com/shiromi-basil/docs-ei/commit/7229cc7a969d3222d31028cb058a5d7881e43964.